src/lib/eZ/settings/fieldtype_services.yaml

```diff
@@ -1,5 +1,5 @@
 parameters:
-    ezrichtext.resources: '%ezpublish.kernel.root_dir%/eZ/Publish/Core/FieldType/RichText/Resources'
+    ezrichtext.resources: 'src/lib/eZ/RichText/Resources'
```

Comment:
Note: bundle has overridden this parameter, used here by integration tests.

Reply:
Could be overridden in tests then?

Reply:
> Could it be overridden in tests then?

Hmm, yeah, maybe for clarity it should, see d05755b. Kept however `ezrichtext.validator.docbook.resource` as default empty because it's used in the file.